pkg/schedule/config/config.go

```diff
@@ -122,6 +122,7 @@ type StoreConfig interface {
 	CheckRegionKeys(uint64, uint64) error
 	IsEnableRegionBucket() bool
 	IsRaftKV2() bool
+	GetRegionBucketSize() uint64
```

Comment:
Is there any way to avoid using this kind of method to get the TiKV config? In case of having more and more config

Reply:
In this pr, this method don't need.